shared/util.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from 'fs'
```

Comment:
rename and rescope to be just about csvOperations. you can put it in shared/util if you like. and move printMembers in to the cabin folder. this isn't a shared utility